test/utils/ActionsRouterPlanner.sol

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
Can we just adapt the Planner.sol file that was already written for posm? Its the same code! To me just another reason that we should get base posm in main.. cause it has some useful V4 integration patterns that hopefully this and router would use :)

Reply:
yeah definitely! just made this because that PR isnt merged yet so needed one here too to test - theyll of course be 1 file whenever theyre both merged